app/controllers/api/v1/purchasers_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::PurchasersController < Api::V1::ApiController
+  before_action :check_company_token
+
+  def show
+    @purchaser = Purchaser.find_by(token: params[:purchaser_token], company: @company)
+    render :show
+  end
+
+  def create
+    params.require(:name)
+    @purchaser = Purchaser.create!(company: @company, name: params[:name])
+    render :show, status: :created
```

Comment:
```suggestion
    render :show, status: :created, location: @purchaser
```

Um header bem legal de enviar na criação é o header de location, o que acham?

Reply:
Obrigado, Henrique. Seguimos sua sugestão e adicionamos ao create!